parm/config/config.vrfy

```diff
@@ -6,7 +6,7 @@
 echo "BEGIN: config.vrfy"
 
 # Get task specific resources
-. $EXPDIR/config.resources vrfy
+. ${EXPDIR}/config.resources vrfy
```

Comment:
```suggestion
. "${EXPDIR}/config.resources" vrfy
```

Reply:
Since we're fixing shellcheck in here. :)

Reply:
I adjusted these quotes as well.